src/web/interface/ConnectionBase.hpp

```diff
@@ -48,7 +48,7 @@ struct ConnectionBase : public util::Taggable {
 public:
     std::string const clientIp;
     bool upgraded = false;
-    bool isAdmin_ = false;
```

Comment:
I'm thinking maybe we should move `isAdmin` to private and add `setAdmin()` to protect it from accidental change?

Reply:
Or maybe make `isAdmin_` a protected member and keep the `isAdmin()`. We should not need to `setAdmin()` from outside of the child classes, right?